From 96506d516ec56082ae5677a0a3f03398173a0556 Mon Sep 17 00:00:00 2001
From: Kai Liu <kaliu@qti.qualcomm.com>
Date: Mon, 15 Jun 2015 13:56:28 +0800
Subject: [PATCH 4/7] cfg80211: Add new wiphy flag WIPHY_FLAG_DFS_OFFLOAD

Add new flag WIPHY_FLAG_DFS_OFFLOAD to the wiphy structure. When this
flag is defined, the driver would handle all the DFS related operations.

CRs-Fixed: 630797
Change-Id: I592722607788dc4e2167c90bb684071cc9fb6986
Signed-off-by: Amar Singhal <asinghal@codeaurora.org>
---
 include/net/cfg80211.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/net/cfg80211.h b/include/net/cfg80211.h
index 21a1014..66e5165 100644
--- a/include/net/cfg80211.h
+++ b/include/net/cfg80211.h
@@ -2627,6 +2627,7 @@ struct cfg80211_ops {
  *	TSPEC sessions (TID aka TSID 0-7) with the NL80211_CMD_ADD_TX_TS
  *	command. Standard IEEE 802.11 TSPEC setup is not yet supported, it
  *	needs to be able to handle Block-Ack agreements and other things.
+ * @WIPHY_FLAG_DFS_OFFLOAD: The driver handles all the DFS related operations.
  */
 enum wiphy_flags {
 	WIPHY_FLAG_SUPPORTS_WMM_ADMISSION	= BIT(0),
@@ -2652,6 +2653,7 @@ enum wiphy_flags {
 	WIPHY_FLAG_HAS_REMAIN_ON_CHANNEL	= BIT(21),
 	WIPHY_FLAG_SUPPORTS_5_10_MHZ		= BIT(22),
 	WIPHY_FLAG_HAS_CHANNEL_SWITCH		= BIT(23),
+	WIPHY_FLAG_DFS_OFFLOAD				= BIT(24),
 };
 
 /**
-- 
1.7.9.5

